docs/source/en/guides/collections.md

```diff
@@ -0,0 +1,187 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# Manage your collections
```

Comment:
I think it may be more consistent to align the doc title with the title in the `toctree`; this makes it easier for users to use the same name to reference the doc

```suggestion
# Collections
```